### PR TITLE
Lm slider

### PR DESCRIPTION
**Pull request by LarryMatte**
added the sliders in different pages and modified the css.

### CODE REVIEW

File: src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss [hunk 1]
```diff
@@ -58,38 +72,40 @@
 }
 
 .slider .prev {
-    left: 20px;
+    left: 0px;
 
 @mixin flip_horizontal();
```
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss]: space

Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss]: indent

File: src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss [hunk 2]
```diff
@@ -58,38 +72,40 @@
...
 
-    cursor: default;
+    cursor: pointer;
 }
 
-.slider .pager a.active {
-    background-position: center bottom;
+.slider .pager a:before {
+    content: "\2022";
+    font-size: 60px;
```
Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss]: font-size, blank line, content

**Comment by jasonlemay** [comment]
LGTM

File: src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss (hunk 2 above)
Comment (LarryMatte) [inline src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss]: @jasonlemay not in the "CSS Rules of Arrangement" file

Comment (jasonlemay) [inline src/main/resources/com/arcbees/website/client/resources/css/ui/slider.gss]: Hey true that, my bad. Pensais que ça allait avec le display

**Pull request merged**